### PR TITLE
Remove dead code

### PR DESCRIPTION
**Pull request by ashtul**

### CODE REVIEW

File: src/aggregate/expr/exprast.c [hunk 1]
```diff
@@ -139,57 +139,10 @@ void RSExpr_Free(RSExpr *e) {
   rm_free(e);
 }
 
-void RSExpr_Print(const RSExpr *e) {
```
Comment (mnunberg) [inline src/aggregate/expr/exprast.c]: This function is used for debugging

File: src/buffer.c [hunk 2]
```diff
@@ -89,15 +89,4 @@ size_t Buffer_WriteAt(BufferWriter *b, size_t offset, void *data, size_t len) {
   size_t sz = Buffer_Write(b, data, len);
   BufferWriter_Seek(b, pos);
   return sz;
-}
-/**
```
Comment (mnunberg) [inline src/buffer.c]: Why not just add a test for this trivial function? I can see this being useful

File: src/document.c [hunk 3]
```diff
@@ -237,15 +237,6 @@ void AddDocumentCtx_Finish(RSAddDocumentCtx *aCtx) {
 // How many bytes in a document to warrant it being tokenized in a separate thread
 #define SELF_EXEC_THRESHOLD 1024
 
-void Document_Dump(const Document *doc) {
```
Comment (mnunberg) [inline src/document.c]: Used for debugging...

File: src/rlookup.c [hunk 4]
```diff
@@ -161,34 +148,6 @@ void RLookupRow_Cleanup(RLookupRow *r) {
...
-  for (const RLookupKey *kk = lk->head; kk; kk = kk->next) {
-    RSValue *vv = RLookup_GetItem(kk, src);
-    if (vv) {
-      RLookup_WriteKey(kk, dst, vv);
-    }
-  }
-  RLookupRow_Wipe(src);
-}
-
-void RLookupRow_Dump(const RLookupRow *rr) {
```
Comment (mnunberg) [inline src/rlookup.c]: Used for debugging...

File: src/value.h [hunk 5]
```diff
@@ -186,18 +184,11 @@ static inline RSValue *RS_ConstStringVal(const char *s, size_t n) {
 /* Wrap a redis string value */
 RSValue *RS_RedisStringVal(RedisModuleString *str);
 
-/**
- *  Create a new value object which increments and owns a reference to the string
- */
-RSValue *RS_OwnRedisStringVal(RedisModuleString *str);
```
Comment (mnunberg) [inline src/value.h]: I can see this being used in the future. If this is dead code, why not add a test for it?

**Comment by ashtul** [comment]
@mnunberg 
Got your point. For now, let's treat this as a todo (or nottodo) list.

**Comment by ashtul** [comment]
Outdated and some were removed in other PRs.

**Pull request closed without merging**